tests/utils_tests/test_numberformat.py

```diff
@@ -57,6 +61,14 @@ def test_decimal_numbers(self):
         self.assertEqual(nformat(Decimal('1234'), '.', grouping=2, thousand_sep=',', force_grouping=True), '12,34')
         self.assertEqual(nformat(Decimal('-1234.33'), '.', decimal_pos=1), '-1234.3')
         self.assertEqual(nformat(Decimal('0.00000001'), '.', decimal_pos=8), '0.00000001')
+        self.assertEqual(nformat(Decimal('9e-19'), '.', decimal_pos=2), '0.00')
+        self.assertEqual(nformat(Decimal('.00000000000099'), '.', decimal_pos=0), '0')
+        self.assertEqual(nformat(Decimal('3.'), '.'), '3')
+        self.assertEqual(nformat(Decimal('3.0'), '.'), '3.0')
+        self.assertEqual(nformat(Decimal('1e16'), '.', thousand_sep=',', grouping=3, force_grouping=True),
```

Comment:
Using hanging indent as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). e.g.
```python
self.assertEqual(
    nformat(Decimal('1e16'), '.', thousand_sep=',', grouping=3, force_grouping=True),
    '10,000,000,000,000,000'
)
```